test/sockjs_client_wrapper_test.dart

```diff
@@ -105,74 +105,74 @@ void _integrationSuite(
   SockJSClient createCorClient(),
   SockJSClient create404Client(),
 ) {
-  SockJSClient client;
+  SockJSClient? client;
```

Comment:
Could we make this `late` instead of nullable to avoid all the `!`s below?

Reply:
I thought about that, but we explicitly make it `null` in the tear down. It seems like we should this so we don't get weird behavior with the client sticking around if we forget to set it in one of the tests.

Reply:
Makes me _sort of_ wish tests were class-based so that you didn't have to deal with scoping that exists beyond the test's lifecycle.